index.html

```diff
@@ -0,0 +1,407 @@
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_forum.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 medium-paragraph-size"
+								id="program-task">
+								Forum
+							</h3>
+
+							<p class="text-light text-small">
+								Have the time to share your thoughts and opinions with experts
+								for each topic.
+							</p>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_workshop.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 medium-paragraph-size"
+								id="program-task">
+								Work shop
+							</h3>
+
+							<p class="text-light text-small">
+								Try creating your own work using open source license rather than
+								just watching at it.
+							</p>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_opportunities.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 medium-paragraph-size"
+								id="program-task">
+								Ignite
+							</h3>
+
+							<p class="text-light text-small">
+								get opportunities to network with CC affiliates around the
+								world, also after the summit.
+							</p>
+						</div>
+					</div>
+					<h4 class="see-program text-center m-1 py-2 paragraph-size">
+						SEE THE WHOLE PROGRAM
+					</h4>
+					<div
+						class="shadow-sm p-2 mb-2 bg-danger mt-4 container"
+						id="join-program">
+						<h4 class="text-center text-white p-1 paragraph-size">
+							Join Victoria Alumni 2023 Summit
+						</h4>
+					</div>
+				</section>
+				<!-- feature speaker -->
+				<section class="feature-speaker-container bgColor-2 py-2">
+					<div class="speaker-content container-fluid">
+						
+							<h4 class="mt-3 text-dark text-center head-font-size">Feature Speaker</h4>
+						
+						<div class="d-flex justify-content-center"><hr class="bar bg-danger"> </hr></div>
+<!-- speakers mobile and desktop -->
+						<div class="feature-speaker-bio-mobile"></div>
+						<div class="feature-speaker-bio"> </div>
+
+						<div class="btn-more">
+							<button type="button" class="btn btn-light text-center textColor-dark w-100 mb-4  paragraph-size" data-bs-toggle="collapse" data-bs-target=".multi-collapse" aria-expanded="false" aria-controls="multiCollapseExample1">more <span class="text-danger"> <i class="fa fa-chevron-down fa-lg"></i> </span></button>
```

Comment:
Great job @ghermaico135 ✋

- [ ] [Optional] Kindly make sure when the more button is clicked, the remaining speakers are rendered and the more button is hidden.
![more](https://user-images.githubusercontent.com/63917725/234874760-25725d26-01cf-457d-91f3-8d2ade8b8ddc.PNG)